applications/source/pom.xml

```diff
@@ -10,26 +10,27 @@
     <packaging>pom</packaging>
 
     <modules>
+        <module>cdc-debezium-source</module>
```

Comment:
I am guessing the only change is adding the new app? I appreciate the alpha-order, but a separate commit would make it easier to see what is formatting vs actual changes.